### DISCUSSION TITLE
.NET 6 Isolated Az Durable Function

### DISCUSSION THREAD

**Discussion by zakkra** [Azure/azure-functions-durable-extension#2301] [Q&A] [answered]
Apparently it is supported:
https://azure.microsoft.com/en-us/updates/public-preview-durable-functions-support-for-isolated-net-worker-process/
but I can't find any samples and just adding durable functions Nuget to an isolated function app gives me 
`Error	AZFW0001	The attribute 'OrchestrationTriggerAttribute' is a WebJobs attribute and not supported in the .NET Worker (Isolated Process).`
Any pointers?
Are durable functions being maintained?

**Comment by cgillum** [comment, marked as answer]
Yes, Durable Functions for .NET Isolated is supported as a preview right now.  The .NET Isolated worker doesn't allow using the "WebJobs" extensions, so a new extension was required.

The team is working on putting together proper documentation for this, but for the time being, you can find instructions for how to get started here: https://github.com/microsoft/durabletask-dotnet#usage-with-azure-functions.
